cycode/cli/main.py

```diff
@@ -156,6 +164,10 @@ def main_cli(context: click.Context, verbose: bool, output: str):
 
     context.obj['output'] = output
 
+    if user_agent:
```

Comment:
why not to send it in all queries?

Reply:
Because here we enrich the user agent with provided information in the CLI option. The default user agent will be sent with each request.

Only if the user agent option is passed the base user agent will be enriched with application information that works with our CLI. I mean our plugins that use CLI under the hood